### PR TITLE
Allow users to enable/disable params

### LINKED ISSUE #28: enable / disable params in mv-params.json
add the ability to enable / disable params in the input mv-params.json

include:
```
"enabled": "true|false"
```
This requires schema to understand the new key "enabled". Multiplex code needs to ignore params with "enabled" key.

### PR DESCRIPTION
**Pull request by rafaelfolco**
Useful for quickly enabling/disabling params from the test.

Closes #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,14 @@ specified with the `args` and `vals` keywords, identical to the "common-params"
 from the `global-options`. Likewise, the `role` key is optional and defaults to 'client'
 if omitted. Valid roles are "client", "server" and "all".
 
+It is also possible to enable/disable params my marking them as "enabled" or "disabled".
+The schema accepts either "yes" and "no" for both keywords, as shown below:
+```
+    { "arg": "rw", "vals": [ "read", "write" ], "enabled": "yes" }
+    { "arg": "rw", "vals": [ "read", "write" ], "enabled": "no" }
+    { "arg": "rw", "vals": [ "read", "write" ], "disabled": "yes" }
+    { "arg": "rw", "vals": [ "read", "write" ], "disabled": "no" }
```
Comment (k-rister) [inline README.md:59]: Remove disabled examples...

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Multiplex requires a JSON file with the following format:
             "name": "common-params",
             "params": [
                 { "arg": "bs", "vals": [ "4k", "8k" ], "role": "client" },
-                { "arg": "rw", "vals": [ "read", "write" ] }
+                { "arg": "rw", "vals": [ "read", "write" ], "disabled": "no" }
```
Comment (k-rister) [inline README.md:23]: Change to "enabled"

**Review by k-rister — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rafaelfolco) [reply README.md]: ack

File: README.md (hunk 1 above)
Comment (rafaelfolco) [reply README.md]: ack

File: README.md [hunk 3]
```diff
@@ -50,6 +50,12 @@ specified with the `args` and `vals` keywords, identical to the "common-params"
 from the `global-options`. Likewise, the `role` key is optional and defaults to 'client'
 if omitted. Valid roles are "client", "server" and "all".
 
+It is also possible to enable/disable params by marking them as "enabled".
+The schema accepts either "yes" and "no" for the "enabled" keyword, as shown below:
+```
+    { "arg": "bs", "vals": [ "4k", "8k" ], "role": "client", "enabled": "no" },
+    { "arg": "rw", "vals": [ "read", "write" ], "enabled": "yes" }
+```
```
Comment (atheurer) [inline README.md:58]: @rafaelfolco @k-rister should this state that "enabled" is optional and if not used the behavior is enabled=yes?

Comment (rafaelfolco) [reply README.md:58]: yep, I can add it.

**Review by k-rister — COMMENTED** [review]
I think this will be good once @atheurer's comment is taken care of.

**Comment by rafaelfolco** [comment]
I think I see a potential problem:
The resulting bench-params.json has now "enabled" key....
I need to change rickshaw schema to accept "enabled" key, or remove enabled from bench-params.json at all.  I vote on the 2nd approach.
Which you prefer @atheurer @k-rister ?

**Comment by k-rister** [comment]
> I think I see a potential problem:
> The resulting bench-params.json has now "enabled" key....
> I need to change rickshaw schema to accept "enabled" key, or remove enabled from bench-params.json at all. I vote on the 2nd approach.
> Which you prefer @atheurer @k-rister ?

While I didn't check for this when reviewing your patches, I think I assumed that the enabled tag would be stripped when generating bench-params.json.

**Review by k-rister — APPROVED** [review]

**Pull request merged**